### DISCUSSION TITLE
GitHub CoPilot Does Nothing

### DISCUSSION THREAD

**Discussion by jmills-billgo** [community/community#6780] [Copilot Conversations]
I've been accepted as a copilot user and I've installed GitHub CoPilot as a VSCode extension on my MacOS. However, nothing happens. When I write code I receive no suggestions. I followed the examples exactly. I have disabled/re-enabled CoPilot as a VSCode extension. I've restarted VSCode. Nothing. As it is, GitHub CoPilot isn't doing anything at all. Yes I've checked VSCode notifications and accepted all agreements. How do you trigger CoPilot to make suggestions? It isn't doing anything at all for me.

**Comment by gravityslave** [comment]
Assumption is you've entered their recommended code line 'function calculateDaysBetweenDates(begin, end) {' into a new JS file? 
then hit CTRL ENTER - to see the list of solutions?  or Command ]?

It didn't autopopulate until I did the commands

**Reply by jmills-billgo** [reply]
I entered that exact line of code. No automatic suggestions were made by CoPilot. I pressed CTRL ENTER and a new pane opened with 0/10 suggestions.

**Reply by gravityslave** [reply]
bummer. and the status icon has a blue background In the lower right? just going through my steps (Mac as well)

**Reply by Maxwell-Thom** [reply]
same here, no icon

**Reply by jmills-billgo** [reply]
The GitHub CoPilot icon is on the bottom right for me. But either way, it has a blue background and has no reported issues.

**Reply by slasho** [reply]
working well here... could be firewall issue? maybe some blocked url ?

**Reply by jmills-billgo** [reply]
Certainly possible, but how might that be determined?

**Reply by jmills-billgo** [reply]
I would think, if a firewall/proxy was causing issues, CoPilot would at least report the issue instead of returning no suggestions.
